packages/components/src/internal/components/domainproperties/dataset/DatasetDesignerPanels.tsx

```diff
@@ -338,7 +373,7 @@ export class DatasetDesignerPanelImpl extends React.PureComponent<Props & Inject
             .catch(error => {
                 console.error(error);
                 setSubmitting(false, () => {
-                    this.props.onComplete(model, resolveErrorMessage(error));
+                    this.setState({importErrorModalOpen: true, importError: error});
```

Comment:
Do we need both of these state properties (i.e. importErrorModalOpen and importError), or can we just use the importError !== undefined to trigger when to open the model?

Reply:
Good point, fixed!